### PR TITLE
REL-2369 demote Pluto

### PR DESCRIPTION
**Pull request by tpolecat**
This PR removes `NamedTarget.Pluto` and adds a `To2016A` conversion that swaps it for a `ConicTarget` with Pluto's orbital elements.

### CODE REVIEW

**Comment by fnussber** [comment]
Poor Pluto... :cry:

**Comment by fnussber** [comment]
On a different note: :+1:

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/SPTargetPio.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
     private static final String _PERIHELION = "perihelion";
     private static final String _EPOCH_OF_PERIHELION = "epochOfPeri";
 
-    private static final DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.FULL);
+    public static final DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.FULL);
```
Comment (tpolecat) [inline bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/SPTargetPio.java]: (Made this public so I could test `Date` equality in terms of the PIO representation.)

**Comment by cquiroz** [comment]
Just want to express my disappointment at having this PR approved, Pluto deserved more :sob: 

nh-pluto-in-true-color_2x_jpeg

**Pull request merged**